spec.md

```diff
@@ -215,6 +215,30 @@ help intermediate gateways determine how to route the events.
     * urn:event:from:myapi/resourse/123
     * mailto:cncf-wg-serverless@lists.cncf.io
 
+### subject
+* Type: `String`
+* Description: This describes the subject of the event in the context of the
+  event producer (identified by `source`). In publish-subscribe scenarios,
+  a subscriber will typically subscribe to events emitted by a `source`,
+  but the `source` identifier alone might not be sufficient as a qualifier for
+  any specific event if the `source` context has internal sub-structure.
+* Example: A subscriber might register interest for when new blobs are created
```

Comment:
In other fields, Example is after the Constrains section.

Reply:
I can shuffle that

Reply:
If the event source has an internal sub-structure, then the event consumer could be interested in any part of the sub-structure. How could the producer know which part of the sub-structure the consumer is interested in, thus which part to put in the subject? Also since it is an internal sub-structure of the source, how could the consumer know how to interpret the semantics of the subject?

Reply:
That is up to the pub/sub infrastructure to provide. In Event Grid we provide filters that let you run a string expression over the subject to filter that down. The publisher doesn’t know, it just publishes for the scope it was asked to deliver events for.